lightning/src/ln/channelmanager.rs

```diff
@@ -110,13 +110,15 @@ pub(super) enum PendingHTLCRouting {
 		payment_metadata: Option<Vec<u8>>,
 		incoming_cltv_expiry: u32, // Used to track when we should expire pending HTLCs that go unclaimed
 		phantom_shared_secret: Option<[u8; 32]>,
+		custom_tlvs: Option<Vec<(u64, Vec<u8>)>>,
 	},
 	ReceiveKeysend {
 		/// This was added in 0.0.116 and will break deserialization on downgrades.
 		payment_data: Option<msgs::FinalOnionHopData>,
 		payment_preimage: PaymentPreimage,
 		payment_metadata: Option<Vec<u8>>,
 		incoming_cltv_expiry: u32, // Used to track when we should expire pending HTLCs that go unclaimed
+		custom_tlvs: Option<Vec<(u64, Vec<u8>)>>,
```

Comment:
optional nit: I know it’s documented elsewhere, but maybe document the tuple here for easy reference.